pldmtool/pldm_platform_cmd.cpp

```diff
@@ -1491,6 +1495,135 @@ class GetPDR : public CommandInterface
         }
     }
 
+    std::string getRdeResourceName(uint8_t* ptr, size_t length)
+    {
+        // Remove null terminator if present
+        if (length > 0 && ptr[length - 1] == '\0')
+        {
+            --length;
+        }
+        // NOLINTNEXTLINE(cppcoreguidelines-pro-type-reinterpret-cast)
+        return std::string(reinterpret_cast<const char*>(ptr), length);
+    }
+
+    std::string getMajorSchemaVersion(ver32_t version)
+    {
+        char version_buffer[1024] = {0};
+        int rc = 0;
+
+        if (version.alpha == 0xFF && version.update == 0xFF &&
+            version.minor == 0xFF && version.major == 0xFF)
+            return std::string("?.?");
+        else
+        {
+            rc = ver2str(&version, version_buffer, sizeof(version_buffer));
+        }
+        return std::string(version_buffer, rc);
+    }
+
+    void printRedfishActionPDR(const uint8_t* data, const uint16_t data_length,
```

Comment:
```suggestion
    void printRedfishActionPDR(const uint8_t* data, const uint16_t dataLength,
```

Please follow lower camel case for the variables in CPP files

Reply:
ACK